### PR TITLE
Hardcoding backlink buttons to change depending on title name to work…

### PR DESCRIPTION
**Pull request by lorenzoAtBEIS**
Hardcoding backlink buttons to change depending on title name to work with Javascript disabled.

### CODE REVIEW

File: INZFS/Pages/EligibilityResult.cshtml [hunk 1]
```diff
@@ -5,12 +5,38 @@
...
+            backLink = "/eligibilityChecker/Question6";
+            break;
+        default: backLink = "/eligibilityChecker/Question1"; break;
+    }
 }
 
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-three-quarters">
-                <a href="#" class="govuk-back-link" id="govuk-back-link">Back</a>
+                <a href=@backLink class="govuk-back-link" id="govuk-back-link">Back</a>
```
Comment (sunilbeis) [inline INZFS/Pages/EligibilityResult.cshtml:39]: Wonder if it should be within a double quote

File: INZFS/Pages/EligibilityChecker.cshtml [hunk 2]
```diff
@@ -71,7 +78,7 @@
                         </div>
                         <div class="govuk-radios__item">
                             <input id="checkNo" class="govuk-radios__input" type="radio" value="0" asp-for=@questionSettingA>
-                            <label class="govuk-label govuk-radios__label" for=""checkNo">
+                            <label class="govuk-label govuk-radios__label" for="" checkNo">
```
Comment (sunilbeis) [inline INZFS/Pages/EligibilityChecker.cshtml:81]: Extra double quote?

File: INZFS/Pages/EligibilityChecker.cshtml [hunk 3]
```diff
@@ -10,33 +10,40 @@
...
         case "Question6":
             questionSettingA = "Question6";
+            backLink = "/eligibilityChecker/Question5";
             break;
         default: questionSettingA = "Question1"; break;
     }
 }
 
-<a href="#" class="govuk-back-link" id="govuk-back-link">Back</a>
+<a href=@backLink class="govuk-back-link" id="govuk-back-link">Back</a>
```
Comment (sunilbeis) [inline INZFS/Pages/EligibilityChecker.cshtml:46]: Wonder if it should be within a double quote

**Review by sunilbeis — CHANGES_REQUESTED** [review]

File: INZFS/Pages/EligibilityResult.cshtml (hunk 1 above)
Comment (lorenzoAtBEIS) [reply INZFS/Pages/EligibilityResult.cshtml]: Thanks man, cosmetic changes applied

File: INZFS/Pages/EligibilityChecker.cshtml (hunk 3 above)
Comment (lorenzoAtBEIS) [reply INZFS/Pages/EligibilityChecker.cshtml]: Changes applied

**Review by sunilbeis — APPROVED** [review]

**Pull request merged**